appsync/emailToAppSync.ts

```diff
@@ -91,8 +91,8 @@ export const handler = async (event: SESEvent) => {
                 subject: r.ses.mail.commonHeaders.subject,
                 timestamp: r.ses.receipt.timestamp,
                 body: {
-                    text,
-                    html: html ? (html as string) : undefined,
+                    text: text ? text : undefined,
```

Comment:
Because you made `text` optional in the `GQLMail` definition above, you can simplify this line:
```suggestion
                    text,
```

Reply:
Thanks for the advice, I've changed that.